frontend/vre/field/record.annotations.view.js

```diff
@@ -39,7 +54,9 @@ export var RecordAnnotationsView = RecordFieldsBaseView.extend({
     },
 
     editEmpty: function() {
-        this.edit(new Backbone.Model());
+        this.edit(new Annotation({
+            "oa:hasTarget": this.collection.target,
```

Comment:
Ah. so that's where you put it. I already wondered why you didn't seem to need a `defaults` method on the `Annotation` model. This is probably a better solution. :+1: